packages/driver/src/cy/commands/local_storage.coffee

```diff
@@ -30,17 +30,20 @@ module.exports = (Commands, Cypress, cy, state, config) ->
       null
 
   Commands.addAll({
-    clearLocalStorage: (keys) ->
+    clearLocalStorage: (keys, options = {}) ->
+      _.defaults options, {log: true}
+      
       ## bail if we have keys and we're not a string and we're not a regexp
       if keys and not _.isString(keys) and not _.isRegExp(keys)
         $errUtils.throwErrByPath("clearLocalStorage.invalid_argument")
 
       remote = clearLocalStorage(state, keys)
 
-      Cypress.log
-        name: "clear ls"
-        snapshot: true
-        end: true
+      if options.log isnt false
+        options._log = Cypress.log
```

Comment:
```suggestion
      if options.log
        Cypress.log
```

How about removing unnecessary `isnt false` and `options._log`?

Reply:
Update in https://github.com/cypress-io/cypress/pull/6495/commits/fc5aacfdbfd00fc584336ec413febc486f2b430e